libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -888,11 +888,8 @@ bool GCS_MAVLINK::send_gps_raw(AP_GPS &gps)
         return false;
     }
 
-    if (gps.highest_supported_status(0) > AP_GPS::GPS_OK_FIX_3D) {
-        if (HAVE_PAYLOAD_SPACE(chan, GPS_RTK)) {
-            gps.send_mavlink_gps_rtk(chan);
-        }
-
+    if (HAVE_PAYLOAD_SPACE(chan, GPS_RTK)) {
```

Comment:
The check could be raised to only send the information when the status is > DGPS, (I realize this drops the message for the case of actually trying to operate DGPS only, but I think thats acceptable.